### DISCUSSION TITLE
unblockability maybe

### DISCUSSION THREAD

**Discussion by RealProthean** [UseInterstellar/Interstellar#312] [Ideas]
it would be cool if you could make interstellar a deployable file, like where you download the file and click on it and it boots up like a google window. this would help prevent schools and businesses from blocking interstellar from the devices, letting people do what they want or need to. just thought i would share this and maybe see if this idea would go anywhere or spark more ideas from others.

**Comment by VillainsRule** [comment]
noted.

**Comment by VillainsRule** [comment]
it's not implemented in IN, but I added this to [annoAPI](https://github.com/AnnotateHQ/annoapi) for site developers.
it will also be added to [annotate](https://annotate.lol) soon.

**Comment by 0mhx** [comment]
This could be possible with a lot of JS changes but the thing is that UV then has to be hosted elsewhere and then that would make it blockable again.

**Comment by Erik-Da-coder** [comment]
Some one made a static uv https://github.com/TheTIW/UV-Static just a fyi

**Comment by Erik-Da-coder** [comment]
I does work too

**Comment by heyoitsame** [comment]
this would help me a lot

**Comment by 0mhx** [comment]
Service Workers, a backend part of IN, requires https. Not really possible.
Also, you would need a bare server, which has to be hosted somewhere and can be blocked.

**Comment by SayanisCool** [comment]
can someone make a proxy with google or to disable securely because the ones I tried didn't work

**Comment by Erik-Da-coder** [comment]
https://github.com/3kh0/ext-remover

**Comment by Erik-Da-coder** [comment]
Got u

**Reply by SayanisCool** [reply]
this one is blocked too :(

**Comment by Erik-Da-coder** [comment]
Wdym

**Comment by SayanisCool** [comment]
when i click it its blocked. <3

**Comment by Erik-Da-coder** [comment]
Sounds like a u problem use a home computer then send to school email

**Comment by SayanisCool** [comment]
my school blocked the emails only teachers can send emails to students and
idk how you did

**Reply by Erik-Da-coder** [reply]
L how r u on GitHub but the other GitHub link works

**Reply by Erik-Da-coder** [reply]
Use this https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://fishme.me/&ved=2ahUKEwi50riH6PGDAxUbFjQIHYqOAZEQFnoECBUQAQ&usg=AOvVaw3j3V2XA1DCkRRGYPTEX7SH

**Comment by Erik-Da-coder** [comment]
Go to the article on tick tock and click on the word  misleading then go to the 404 link on that page

**Comment by Erik-Da-coder** [comment]
It is a unblocker

**Comment by SayanisCool** [comment]
thats blocked

**Comment by Erik-Da-coder** [comment]
L sry

**Comment by xbubbo** [comment]
Interstellar is not going to be usable as a file anytime soon, this would result in alot of code changes and hosting parts of the backend on other domains which would cause it to be blocked still.

Consider [Self-Hosting](https://github.com/interstellarnetwork/interstellar?tab=readme-ov-file#deployment).
